src/Valet.UnitTests/Services/DockerServiceTests.cs

```diff
@@ -277,4 +277,111 @@ public void VerifyImagePresentAsync_NotPresent_ThrowsException()
         Assert.ThrowsAsync<Exception>(() => _dockerService.VerifyImagePresentAsync(image, server, version));
         _processService.VerifyAll();
     }
+
+    [Test]
+    public void GetCurrentImageDigest_ParsesDigestCorrectly()
+    {
+        // Arrange
+        var image = "valet-customers/valet-cli";
+        var server = "ghcr.io";
+
+        _processService.Setup(handler =>
+            handler.RunAndCaptureAsync(
+                "docker",
+                $"image inspect --format={{{{.Id}}}} {server}/{image}:latest",
+                It.IsAny<string?>(),
+                It.IsAny<IEnumerable<(string, string)>?>(),
+                It.IsAny<bool>()
+            )
+        ).Returns(Task.FromResult("sha256:67eed1493c461efd993be9777598a456562f4e0c6b0bddcb19d819220a06dd4b"));
+
+        // Act
+        var result = _dockerService.GetCurrentImageDigestAsync(image, server);
```

Comment:
```suggestion
    public async Task GetCurrentImageDigest_ParsesDigestCorrectly()
    {
        // Arrange
        var image = "valet-customers/valet-cli";
        var server = "ghcr.io";

        _processService.Setup(handler =>
            handler.RunAndCaptureAsync(
                "docker",
                $"image inspect --format={{{{.Id}}}} {server}/{image}:latest",
                It.IsAny<string?>(),
                It.IsAny<IEnumerable<(string, string)>?>(),
                It.IsAny<bool>()
            )
        ).Returns(Task.FromResult("sha256:67eed1493c461efd993be9777598a456562f4e0c6b0bddcb19d819220a06dd4b"));

        // Act
        var result = await _dockerService.GetCurrentImageDigestAsync(image, server);
```

This should use async/await instead of using `Task.Result`